doc/en/user/source/production/container.rst

```diff
@@ -52,4 +52,29 @@ For more information about JVM configuration, see the article `Performance tunin
       uintx InitialHeapSize   := 268435456     {product}
       uintx MaxHeapSize       := 4294967296    {product}
     
-   The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
\ No newline at end of file
+   The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
+   
+Enable CORS
+-----------
+
+The standalone distributions of GeoServer use an application server include the Jetty application server. Enable Cross-Origin Resource Sharing (CORS) to allow JavaScript applications outside of your own domain to use GeoServer.
```

Comment:
Suggest "The standalone distributions of GeoServer use an application server." or "The standalone distributions of GeoServer include the Jetty application server." in place of the first sentence, which doesn't make sense to me as written.